components/ide/jetbrains/gateway-plugin/doc/installing.md

```diff
@@ -1,18 +1,5 @@
 ## Installing Gitpod plugin in JetBrains Gateway
 
-<!-- Uncomment when nightly channel is available
-- Start JetBrains Gateway
-- [Add a custom repository](https://www.jetbrains.com/help/idea/managing-plugins.html#add_plugin_repos)
-  for the nightly channel: https://plugins.jetbrains.com/plugins/list?channel=nightly&pluginId=18438-gitpod-gateway
-- Install `Gitpod Gateway`
-  [from the marketplace](https://www.jetbrains.com/help/idea/managing-plugins.html#install_plugin_from_repo)
-- Restart JetBrains Gateway
--->
-
-- Download the plugin archive: https://plugins.jetbrains.com/plugin/download?rel=true&updateId=154782
-    - We are in the process of establishing update channels on JetBrains marketplace to enable auto upgrade. As for now
-      if something does not work please make sure to report an issue and check here for newer version of the plugin.
-- Start JetBrains Gateway
-- [Install from the plugin archive](https://www.jetbrains.com/help/idea/managing-plugins.html#install_plugin_from_disk)
-  - You may see an error about a missing dependency, ignore it. We are in process of sorting it out.
-- Press `Apply` and then restart JetBrains Gateway
+- Install latest [JetBrains Gateway](https://www.jetbrains.com/remote-development/gateway/)
```

Comment:
Question (out of curiosity):

Any reason to _not_ suggest toolbox? As opposed to direct gateway download?

Reply:
Ah, no, I thought it is referencing toolbox there :D

Reply:
I think it is alright actually. Let's decide JB how to guide a user to installation. As far as I understand Gateway can self update.